### DISCUSSION TITLE
Library scan progress

### DISCUSSION THREAD

**Discussion by onikienko** [quodlibet/quodlibet#3865] [Q&A]
My music library is located on the network drive. My lib is relatively big - about 50K tracks (~450GB).
So I installed QuodLibet on my Mac, defined the path to my lib located on a network drive. And I see nothing. I understand that scanning may take time but I expect to see some UI element indicating that scanning is in progress. But I see nothing. 
No errors. No progress. No tracks. 

Am I doing something wrong? Or player does not support network drives? I am on Mac air m1.

**Comment by bassstorm** [comment]
Hello!
Nope, that's just how QL works. You may find relevant discussion in #3042 comments.
But I'm not quite sure from your question, did library scan end successfully after a while? If not, it's definitely an issue.

P.S. For me, scan (over WiFi) of library of your size takes literally hours. But results are partially visible (and searchable) during the process.
